scripts/seed-data

```diff
@@ -18,4 +18,8 @@ cloud_init:
     - file:/var/lib/rancher/conf/$(basename ${CLOUD_DATA})
 EOF
 
+for f in ${FILES[@]}; do
+  cp -a ${f} ${BASE_DIR}/lib/rancher/conf/
```

Comment:
My intent was just to drop all the specified files (and dirs) to `/var/lib/rancher/conf/`.
For a second I've had an idea of letting to drop files anywhere, but then I thought: YAGNI